src/FffGcodeWriter.cpp

```diff
@@ -2524,22 +2523,22 @@ bool FffGcodeWriter::processSupportInfill(const SliceDataStorage& storage, Layer
                                    support_line_width, support_line_distance_here, current_support_infill_overlap - (density_idx == max_density_idx ? 0 : wall_line_count * support_line_width),
```

Comment:
Dubbel check if the support infill overlap is correct. This infill overlap is also used when generating the wall toolpaths and the contour for the lines (used here) is generated from that "offset" inner_contour.